index.js

```diff
@@ -44,6 +44,7 @@ function requestbody(opts) {
   opts.formLimit = 'formLimit' in opts ? opts.formLimit : '56kb';
   opts.queryString = 'queryString' in opts ? opts.queryString : null;
   opts.formidable = 'formidable' in opts ? opts.formidable : {};
+  opts.returnRawBody = 'returnRawBody' in opts ? opts.returnRawBody : false
```

Comment:
Sorry for the delay @thuitaw but I just thought of an issue... This option makes me think that the raw body for any type of payload would be returned (e.g. for JSON we'd have a string). What do you think about making this option for for all types of requests?

Reply:
@MarkHerhold I get your point. Let me look into how exactly I'd implement that. I'll get back to you with my findings

Reply:
@MarkHerhold Have updated the returnRawBody parameter to be passed in for all types. Still working on the tests